api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/handler/AvailabilityHandler.java

```diff
@@ -466,4 +471,126 @@ public void getMetricStats(
                 .map(ApiUtils::collectionToResponse)
                 .subscribe(asyncResponse::resume, t -> asyncResponse.resume(serverError(t)));
     }
+
+    @GET
+    @Path("/aggregated")
```

Comment:
This path is not good. Aggregated will be used in a different context soon. Can you put this functionality under `/stats`?